src/main/scala/play/dev/filewatch/PollingFileWatchService.scala

```diff
@@ -43,6 +43,11 @@ object SourceModificationWatch {
 
   @tailrec def watch(sourcesFinder: PathFinder, pollDelayMillis: Int, state: WatchState)(terminationCondition: => Boolean): (Boolean, WatchState) =
     {
+      if (pollDelayMillis < 1000) {
```

Comment:
Hmm. Actually I'm not sure this is the best fix. We really want to poll more often. We don't want to wait 1s before picking up a change